src/AutoBuild.jl

```diff
@@ -1207,13 +1221,18 @@ function build_jll_package(src_name::String, build_version::VersionNumber, code_
 
         # Extract all platforms
         artifacts = Pkg.Artifacts.load_artifacts_toml(artifacts_toml; pkg_uuid=$(repr(jll_uuid("$(src_name)_jll"))))
-        platforms = [Pkg.Artifacts.unpack_platform(e, $(repr(src_name)), artifacts_toml) for e in artifacts[$(repr(src_name))]]
+        if artifacts[$(repr(src_name))] isa Dict
+            # If this is a dictionary, the artifact is platform-independent
+            best_platform = "any"
+        else
+            platforms = [Pkg.Artifacts.unpack_platform(e, $(repr(src_name)), artifacts_toml) for e in artifacts[$(repr(src_name))]]
 
-        # Filter platforms based on what wrappers we've generated on-disk
-        filter!(p -> isfile(joinpath(@__DIR__, "wrappers", replace(triplet(p), "arm-" => "armv7l-") * ".jl")), platforms)
+            # Filter platforms based on what wrappers we've generated on-disk
+            filter!(p -> isfile(joinpath(@__DIR__, "wrappers", replace(triplet(p), "arm-" => "armv7l-") * ".jl")), platforms)
 
-        # From the available options, choose the best platform
-        best_platform = select_platform(Dict(p => triplet(p) for p in platforms))
+            # From the available options, choose the best platform
+            best_platform = select_platform(Dict(p => triplet(p) for p in platforms))
+        end
```

Comment:
The ideal solution here would be to directly write
```julia
include(joinpath(@__DIR__, "wrappers", "any.jl"))
```
for `AnyPlatform` and what was here otherwise, but in this function we don't know the list of platforms and I'd like to avoid breaking registration again :upside_down_face:

Reply:
It will be illegal to have a mixture of `AnyPlatform` and anything else; so I think we do know the list of platforms here.  If it's not `AnyPlatform`, do the normal thing.  If it is `AnyPlatform`, write out only the one file.

Reply:
Of course I was wrong :upside_down_face: